### PR TITLE
Add new image position field and necessary styling to Image backgroun…

### PR DESCRIPTION
**Pull request by petrillek**
I've created this update to allow us to position background images.

TBD:
As this is adding new field we probably need to provide an update function.

### CODE REVIEW

File: modules/modifiers_bg_image/src/Plugin/modifiers/ImageBgModifier.php [hunk 1]
```diff
@@ -44,6 +44,11 @@ public static function modification($selector, array $config) {
         }
       }
```
Comment (kybersoft) [inline modules/modifiers_bg_image/src/Plugin/modifiers/ImageBgModifier.php]: Please remove this empty line :-)

File: modules/modifiers_bg_image/src/Plugin/modifiers/ImageBgModifier.php [hunk 2]
```diff
@@ -44,6 +44,11 @@ public static function modification($selector, array $config) {
...
 
+      if (!empty($config['image_position'])) {
+        $position = str_replace('-', ' ', $config['image_position']);
+        $css['all'][$selector][] = 'background-position:' . $position;
```
Comment (kybersoft) [inline modules/modifiers_bg_image/src/Plugin/modifiers/ImageBgModifier.php]: We should use `$media` instead of `all`.

**Review by kybersoft — CHANGES_REQUESTED** [review]
I think you have bad path for first file inside commit `docroot/sites/start/config/field.field.paragraph.image_modifier.field_image_position.yml` and I pointed on few requested changes - could you please fix them?

**Comment by petrillek** [comment]
I added test for the new code and fixed a small bug in the module.

**Review by kybersoft — APPROVED** [review]
Nice work, thank you Petr!

**Pull request merged**